src/components/add-form/styles.css

```diff
@@ -1,20 +1,37 @@
+.addform-container {
+  height: 100vh;
+  width: 300px;
+}
+
+.addform-container-label {
+  font-size: 16px;
+  font-weight: 700;
+}
+
 .input-item {
-  border-radius: 4px;
+  border-radius: 8px;
   line-height: 2;
   margin-top: 8px;
   margin-bottom: 20px;
-  padding: 0px 10px;
+  padding: 5px 10px;
   width: 100%;
+  border: #f2b807 solid 2px;
 }
 
 .time-frame {
   align-items: flex-start;
   display: flex;
   flex-direction: column;
+  justify-content: space-around;
   font-size: 15px;
```

Comment:
We can delete this line, you are resetting the font size on line 33

Reply:
Done :)